base/files/file_util_starboard.cc

```diff
@@ -316,8 +321,15 @@ bool CreateDirectoryAndGetError(const FilePath &full_path, File::Error* error) {
   AssertBlockingAllowed();
 
   // Fast-path: can the full path be resolved from the full path?
+  int retval;
+  #ifdef _WIN32
+    retval = _mkdir(full_path.value().c_str());
```

Comment:
We shouldn't be using the win32 APIs. 
Can you add a posix_emu thing that provide `mkdir`?

Reply:
Fixed